kinto/core/api.py

```diff
@@ -69,3 +106,19 @@ def default_op_ids(self, service, method):
         op_id = "{}_{}".format(method, resource)
 
         return op_id
+
+    def default_security(self, service, method):
+        """Provides OpenAPI security properties based on kinto policies."""
+
+        definitions = service.definitions
+
+        # Get method view arguments
+        for definition in definitions:
+            met, view, args = definition
+            if met == method:
+                break
```

Comment:
This is never supposed to happen. The ``method`` that is passed to the ``default_security`` is actually fetched the same way that is done here (by inspecting service definitions), but the interface for default_xxx calls on cornice swagger is designed to take only ``service`` as a parameters (so we can have the same interface for all defaults), but we can change the api to take a method definition as an optional argument.